test/rules/iterable_contains_unrelated_type.dart

```diff
@@ -133,22 +134,32 @@ bool takesIterable2(Iterable<String> iterable) => iterable.contains('a'); // OK
 bool takesIterable3(Iterable iterable) => iterable.contains('a'); // OK
 
 abstract class A implements Iterable<int> {}
+
 abstract class B extends A {}
+
 bool takesB(B b) => b.contains('a'); // LINT
 
 abstract class A1 implements Iterable<String> {}
+
 abstract class B1 extends A1 {}
+
 bool takesB1(B1 b) => b.contains('a'); // OK
 
 abstract class A3 implements Iterable {}
+
 abstract class B3 extends A3 {}
+
 bool takesB3(B3 b) => b.contains('a'); // OK
 
 abstract class A2 implements Iterable<String> {}
+
 abstract class B2 extends A2 {}
+
 bool takesB2(B2 b) => b.contains('a'); // OK
 
-abstract class SomeIterable<E> implements Iterable<E> {}
+abstract class AddlInterface {}
+
+abstract class SomeIterable<E> implements Iterable<E>, AddlInterface {}
```

Comment:
While investigating `_findImplementedInterfaces`, trying to figure out how it works, I couldn't find any test cases where the accumulator ever contains more than a single type. There were no commas in this test before this change, so I thought it was a good little addition to land.